src/EnergyPlus/PluginManager.hh

```diff
@@ -243,6 +245,9 @@ struct PluginManagerData : BaseGlobalStruct
     bool apiErrorFlag = false;
     std::vector<std::string> const objectsToFind = {
         "PythonPlugin:OutputVariable", "PythonPlugin:SearchPaths", "PythonPlugin:Instance", "PythonPlugin:Variables", "PythonPlugin:TrendVariable"};
+
+    bool eplusRunningViaPythonAPI = false;
```

Comment:
Interesting.  This isn't code that is commonly touched, so I'm not going to rigor over an extra variable, but I am a bit unsure why you can't just always use the one from state, and not have one on the plugin manager at all.  I guess it's because the plugin manager dtor doesn't take a state argument...so there is no state to access, only local members.